Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -736,6 +736,8 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     if (!isLoaded())
         return;
 
+    extensionController()->sendToAllProcesses(Messages::WebExtensionContextProxy::UpdateGrantedPermissions(m_grantedPermissions), identifier());
```

Comment:
Will this get sent to the processes for other extensions? Or will it only be to the processes for this extension?